File: Codigo_Fonte_c/Lista 2/lista2_3.c
//Exibir a soma de todos os 80 primeiros números inteiros (1+2+3 ... 78+79+80) .
#include <stdio.h>

int main() {
	int i, soma_x;
	i = 1;
	soma_x = 0;

// estrutura de repetição while
	while (i <= 80){
		soma_x = soma_x + i;
		i++;
	}
	printf("A soma dos 80 primeiros termos: %d\n", soma_x);


// estrutura de repetição do while
	i = 1;
	soma_x = 0;
	do
	{
		soma_x = soma_x + i;
		i++;
	}while (i <= 80);
	printf("A soma dos 80 primeiros termos: %d\n", soma_x);


// estrutura de repetição for
	soma_x = 0;
	for(i = 1; i <= 80; i++){
		soma_x = soma_x + i;
	}
	printf("A soma dos 80 primeiros termos: %d\n", soma_x);
	
	return 0;
}

File: Codigo_Fonte_pascal/lista2/lista2_3.pas
program soma_sequencia;

var
   i, soma_x : integer;
begin
     //writeln('Estrutura de repetição while...do');
     i := 1;
     soma_x := 0;
     while i <= 80 do
     begin
          //writeln(i);
          soma_x := soma_x + i;
          i := i + 1;
     end;
     writeln('A soma dos 80 primeiros termos: ', soma_x);
     writeln;

     //writeln('Estrutura de repetição repeat...until');
     i := 1;
     soma_x := 0;
     repeat
           //writeln(i);
           soma_x := soma_x + i;
           i := i + 1;
     until i > 80;
     writeln('A soma dos 80 primeiros termos: ', soma_x);
     writeln;

     //writeln('Estrutura de repetição for');
     soma_x := 0;
     for i := 1 to 80 do
     begin
          //writeln(i);
          soma_x := soma_x + i;
     end;
     writeln('A soma dos 80 primeiros termos: ', soma_x);
     readln;

end.
